css/button.css

```diff
@@ -0,0 +1,97 @@
+:root {
+  --rb-primary: #d97706;
+  --rb-very-light-grey: #f9fafb;
+  --rb-light-black: #111827;
+  --rb-grey: #d1d5db;
+}
```

Comment:
Best to keep all colors together. Tomorrow if you decided to add a new theme, Updating colors in one place will be easier.

Reply:
Solved